server/covmanager/views.py

```diff
@@ -58,8 +58,15 @@ def collections_diff(request):
 
 
 def collections_reportsummary(request, collectionid):
+    collection = get_object_or_404(Collection, pk=collectionid)
+
+    collection_data = {
+        "id" : collection.id,
+        "created" : collection.created.isoformat()
```

Comment:
By default, this returns a datetime string in the format Nov. 21, 2024, 9:48 p.m to the frontend, which is invalid when used with new Date(datetime) on the frontend. Consequently, the formatting helper does not support this string. This update addresses that issue.

Alternatively, we could pass the collection object through the CollectionSerializer (while maintaining either a functional or class-based view) to ensure the date is formatted as an ISO string, which is compatible with the frontend.

Please let us know if this approach would be preferred.

Reply:
This is good, but I would prefer to use the existing serializer instead as you suggest.

Reply:
Alright @jschwartzentruber. We are on it.

Reply:
@jschwartzentruber  We've resolved this comment.